app/views/items/_item.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @item.each do |f| %>
```

Comment:
The general convention is that this would display a single item from a local variable (like `item`), instead of using an instance variable `@items` which was set in the controller and holds a collection of items.

This also would allow the partial to be re-used for anywhere you need to display an item (such as an `items#show` view).